.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Integration and Unit tests
+concurrency: unity_run
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Run Unity tests
+    runs-on: macos-latest
```

Comment:
Need to add support for windows testing as well. I will share the steps with you separately.

Reply:
@crimsonred so we figured out, that this can't be done on GitHub Action container with Windows Server on it.

Reply:
yep :(